layouts/partials/footer_links/suffix.html

```diff
@@ -0,0 +1,3 @@
+<script id="event-short-list" type="text/html">
```

Comment:
Do we need this on all pages? If not, I would recommend adding it to a different layout page such as https://github.com/openhwgroup/openhwgroup.org/blob/master/layouts/index.html

Reply:
Moved it to layouts/index.html